app/utils/formatters.ts

```diff
@@ -0,0 +1,24 @@
+import { format, parseISO } from "date-fns";
+
+// const formatCostToDollars = (cost: number | null) =>
```

Comment:
i left this here because we'll need to render comp costs on the location details pages